src/compile/selection/legends.ts

```diff
@@ -72,7 +72,7 @@ const legendBindings: SelectionCompiler<'point'> = {
           ...(!selCmpt.init ? {value: null} : {}),
           on: [
             // Legend entries do not store values, so we need to walk the scenegraph to the symbol datum.
-            {events, update: 'datum.value || item().items[0].items[0].datum.value', force: true},
+            {events, update: 'datum.value != null? datum.value: item().items[0].items[0].datum.value', force: true},
```

Comment:
Let's use

```suggestion
            {events, update: 'datum.value ?? datum.value: item().items[0].items[0].datum.value', force: true},
```

it's supported in all modern browser and node: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
I wasn't sure that it was supported in the Vega expression language. Does the expression interpreter handle it?

Reply:
Oh, good question. 

Evidently not: [Open the Chart in the Vega Editor](https://vega.github.io/editor/#/url/vega/N4IgJAzgxgFgpgWwIYgFwhgF0wBwqgegIDc4BzJAOjIEtMYBXAI0poHsDp5kTykSArJQBWENgDsQAGhAQaZcUgA2ENAG1QihHDQhMcCJmkga4uroCMAJgAEAfjs2AzCAC+AXVdA).

What you wrote looks good to me then.